### ISSUE TITLE
Showcase v2 Feedback

### ISSUE THREAD

**Issue by ndiego** [WordPress/wporg-showcase-2022#234]
Copied over from WP Slack: https://wordpress.slack.com/archives/C04U953K77A/p1696925380920099?thread_ts=1696525287.393319&cid=C04U953K77A

- As a user, I don't really understand what the links on the left of the 'homepage' banner are/mean. See attachment.
  - What is "Meta Newsroom"? Why is there a Facebook link? Why would I click 'Business' / where is this taking me? We're assuming context that the user doesn't have. Can we rework these into a simple paragraph?
- Canonical URL on search pages uses the 'pretty' URL format; it should use the ?s= format.
- Archive pages (including search results and tags) don't have a H1, so it's not clear what I'm looking at (nb, this should also consider the paginated state).
- Archive pages assume that the user knows where they are / what they're looking at. https://wordpress.org/showcase-v2/browse/ has no context/information about what I'm browsing / what these filters are. Descriptive + helpful introductory on all pages/templates pls :smile:
- Can we kill/hide all tags with fewer than 3(?) members? E.g., https://wordpress.org/showcase-v2/tag/space/ isn't useful :laughing:
- Page <title>s are inconsistent about whether they include (or end in)  "WordPress Showcase"; can we define and use a consistent pattern throughout which includes this?
- I'm also not clear on the difference between categories and tags (good old WordPress, eh? :wink:)
- In the breadcrumb, we have 'Home' immediately below 'Showcase'; can we remove 'Home' (as it doesn't link to the homepage), and have Home > Showcase > etc? Or if that's contentious, just replace 'Home' with 'Showcase' in the breadcrumbs.
- The Next/Previous nav on individual items is confusing, and a worse experience than the 'Related sites' nav. Can we remove it?
- Is there a reason that we're setting noreferrer on links to sites?

**Comment by ndiego** [comment]
cc @jasmussen @marko-srb

**Comment by jasmussen** [comment]
Good stuff here. Marko is a bit busy at the moment so I'll respond to the best of my ability. In general, agree with most of this, so the stuff I didn't comment on/add context to below, would be good to just run with.

> What is "Meta Newsroom"? Why is there a Facebook link? Why would I click 'Business' / where is this taking me? We're assuming context that the user doesn't have. Can we rework these into a simple paragraph?

Conceptually it's the feature version of this box;

![Screenshot 2023-10-10 at 13 51 17](https://github.com/WordPress/wporg-showcase-2022/assets/1204802/c33fe2d4-52cc-4a3d-a453-02682e1a691a)

At the same time, it should remain minimal. We could look to adding icons to denote the category and URL, and in this case, "Meta Newsroom" is the title. The fact that Facebook rebranded to Meta is a point of confusion that's harder for us to fix.

> Archive pages (including search results and tags) don't have a H1, so it's not clear what I'm looking at (nb, this should also consider the paginated state).

I think Kelly brought this up, and there should be a separate issue to just add those titles. Let's do that.

> Archive pages assume that the user knows where they are / what they're looking at. https://wordpress.org/showcase-v2/browse/ has no context/information about what I'm browsing / what these filters are. Descriptive + helpful introductory on all pages/templates pls 😄

Perhaps there's a better example of the general feedback about descriptive introductions, but the feedback doesn't feel entirely valid for the Browse page, as there are both breadcrumbs and a clear title on the page. If we need to add a short paragraph below, that won't be the end of the world, I could just use some clarification on the ask.

> Can we kill/hide all tags with fewer than 3(?) members? E.g., https://wordpress.org/showcase-v2/tag/space/ isn't useful 😆

Agree. 

> In the breadcrumb, we have 'Home' immediately below 'Showcase'; can we remove 'Home' (as it doesn't link to the homepage), and have Home > Showcase > etc? Or if that's contentious, just replace 'Home' with 'Showcase' in the breadcrumbs.

[I feel like this issue captures the problem](https://github.com/WordPress/wporg-mu-plugins/issues/468), which to be clear is an important one to solve. Home should link to the landing page for Showcase, but it should be de-emphasized and as was suggested in the design on that issue, probably be a unified bar, because Showcase will also have a top level item. CC: @fcoveram — we've discussed here a "compact" mode for the two navigation bars that merge them into one for these cases.

> The Next/Previous nav on individual items is confusing, and a worse experience than the 'Related sites' nav. Can we remove it?

Assuming this is referring to these:

![Screenshot 2023-10-10 at 13 59 30](https://github.com/WordPress/wporg-showcase-2022/assets/1204802/0d798f63-a831-4480-a5f5-2ba629676fc7)

I don't mind them. I also think we can remove them if need be. No strong opinion here.

**Comment by StevenDufresne** [comment]
> As a user, I don't really understand what the links on the left of the 'homepage' banner are/mean

I brought that up in the initial review. Here is [context](https://github.com/WordPress/wporg-showcase-2022/pull/145#issuecomment-1679901688).

**Comment by ndiego** [comment]
> Canonical URL on search pages uses the 'pretty' URL format; it should use the ?s= format.

@jonoalderson would you be able to elaborate on this one? The search pages do appear to use the `?s=` format, for example: https://wordpress.org/showcase-v2/?s=space

**Comment by jonoalderson** [comment]
See the canonical URL tag. Actually, given that the page is noindex'd, the canonical URL tag should be removed entirely.

**Comment by marko-srb** [comment]
👋 Few conclusions: 


- Words as meta data don't work here for NYC Company and Orchestre Metropolitan. In case of The New York City Company, we could short the title, if appropriate, as in screenshot. But in case of OM, there's hardly anything we can do for website link. See screenshot. All other entries on hero (4 of them) are fitting into each responsive screen if we add these word labels.
- We had feedback that this hero section on landing should visually reflect as much to the one on single post meta data. This means introducing of the icons would be not applicable, or if we introduce them - they should be added to the single page meta-data as well.

<img width="722" alt="Screen Shot 2023-10-12 at 12 35 49" src="https://github.com/WordPress/wporg-showcase-2022/assets/111751264/2df44315-ea7b-48e4-b48b-3503646b2b34">


Solution that we can do is replace the Orchestre Metropolitan with a shorter name (different website from showcase) and add the wording labels on the left. 

What do you think is good to do here?

**Comment by jonoalderson** [comment]
I don't think it makes sense to prioritize visual consistency between the banner and the single post, as they serve different purposes. It may well make sense to use a completely different visual treatment, have different content, etc, for people who're seeing the Showcase homepage (and saying "what on earth am I looking at?"), vs those who navigate to or land directly on an individual Showcase item where the context is very different.

**Comment by StevenDufresne** [comment]
> What do you think is good to do here?

I agree that section is a bit of a missed opportunity. 

A couple additional points being:
- The second link on the page (site url) navigates the user away from the showcase. Probably not a user path we want to promote.
- The category seems pretty irrelevant. Do we want them to click that?
- Too many clickable options.

Regardless, I think we can launch and track how this section is affecting user behavior and react accordingly.
- Are they clicking the title?
- Are they clicking the image?
- Do they immediately click the website's URL?
- Do they do none of this?

I think we agree that this can be improved but probably doesn't need to be considered a blocker for launch. 

When we come back to iterate and build on the idea that users _should be able to see themselves and their endeavors in the showcase_, we can leverage that section to communicate that and build out the content of the featured sites to read more like use-cases.

## Example
Something to this effect with better treatment and content :).

<img width="1457" alt="Screenshot 2023-10-13 at 12 03 36 PM" src="https://github.com/WordPress/wporg-showcase-2022/assets/1657336/b5713a51-a1db-4b0f-b29c-7f7712c0ccfe">

**Comment by jonoalderson** [comment]
>I think we agree that this can be improved but probably doesn't need to be considered a blocker for launch.
I think we're close, but I'm nervous that we're not hitting the mark on "_does the homepage of this thing make sense, clearly convey the right message, and help users get to the right place_". 

We get one chance to launch this and make a song and dance. Instead of spending weeks measuring (unsurprisingly few?) interactions with what we know is a confusing interface, I'd vote that we spend just a tiny bit more time and reinvent this key component before shipping it.

That transformation to a statement is _much_ more intuitive, and I don't think there's a huge amount of extra work needed to do something more meaningful in that space than the box of links.

**Comment by ndiego** [comment]
For the sake of ideas, here is another: 

<img width="1681" alt="image" src="https://github.com/WordPress/wporg-showcase-2022/assets/4832319/b88d31fc-2bfb-4a5c-b311-d554ffc5870e">

- The spotlighted site name is larger and no longer a link
- The category taxonomy remains, but is more inline with how it displays in the grid below
- The direct link to the site is removed
- There is now a clear CTA to visit the individual page for the spotlighted site

**Comment by pablohoneyhoney** [comment]
I agree that this UI could be slightly more custom. And I like where your exploration is going @ndiego 

I'm not entirely sure if we should hide the url because some instances might be a sub-site of the mother company (let's say Wirecutter within NYT) which would get the titles too long? And perhaps the url is more specific. Even if it's less prominent. Plus it gives a way to open in new tab the real site. 

I'm not sure about the main CTA above (if it's intended to direct to the real site). If it's meant to go to the site detail page, is the link/button label clear? is it _Learn more_? Not a strong opinion, but because I don't see the url, it feels I'm going to the site itself.
